### PR TITLE
Typescript typings

### PR DESCRIPTION
**Pull request by iRoachie**
Typescript typings for those wanting to use react-native with typescript

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+// TypeScript Version: 2.2.2
+
+import React, { Component } from 'react'
+import { ImageURISource } from 'react-native'
+
+interface SvgUriProps {
+    /**
+     * The width of the rendered svg
+     */
+    width: number | string
```
Comment (ligaz) [inline index.d.ts]: This should be optional.

File: index.d.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    /**
+     * The height of the rendered svg
+     */
+    height: number | string
```
Comment (ligaz) [inline index.d.ts]: This should be optional.

File: index.d.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+    /**
+     * Source path for the .svg file
+     * Expects a require('path') to the file or object with uri.
+     * e.g. source={require('my-path')}
+     * e.g. source={{ur: 'my-path'}}
+     */
+    source: ImageURISource
```
Comment (ligaz) [inline index.d.ts]: This should be optional. It is possible to pass just the `svgXmlData`.

**Comment by iRoachie** [comment]
@ligaz Updated to be optional

**Comment by ligaz** [comment]
👍

**Comment by iRoachie** [comment]
@ligaz Merge?

**Comment by ligaz** [comment]
@iRoachie I do not have an ownership of this repo so I'm unable to merge it. I do maintain my own fork though and will include your typings there.

**Comment by matc4** [comment]
Merged!

**Comment by iRoachie** [comment]
@matc4 Version bump?

**Comment by iRoachie** [comment]
@matc4 Did you publish new version to npm?

**Comment by matc4** [comment]
Hello @iRoachie , not yet, i am merging others PR's to send a new version with all the new features.

**Comment by iRoachie** [comment]
Thanks 👍

**Pull request merged**